src/components/Cell/Cell.tsx

```diff
@@ -42,14 +53,19 @@ const Cell = ({id, toggleModal}: GridCell) => {
         backgroundColor: 'LawnGreen'
     }
 
-    const divStyle = {
+    const divStyle = !isDisabled && {
```

Comment:
nice job applying the border to not disabled cells. the minor bug where disabled cells would have a white border is no longer occurring.